djangoProject/djangoProject/settings.py

```diff
@@ -75,6 +75,34 @@
 
 WSGI_APPLICATION = 'djangoProject.wsgi.application'
 
+LOGGING = {
+    'version': 1,
+    'disable_existing_loggers': False,
+    'formatters': {
+        'standard': {
+            'format': '%(asctime)s %(levelname)s %(name)s %(message)s'
```

Comment:
Consider adding the `%(module)s` and `%(funcName)s` fields if you want more detailed information about the module or function generating the log:
```
'format': '%(asctime)s %(levelname)s %(name)s [%(module)s:%(funcName)s] %(message)s'
```